src/server.js

```diff
@@ -42,12 +45,15 @@ const listen = (port) => {
 };
 
 const port = process.env.PORT || 3002;
-if (port > 3002) {
-  listen(port);
-} else {
-  findPort(port, port + 100, function(ports) {
-    return listen(ports[0]);
-  });
-}
+
+cluster(function() {
```

Comment:
RealTalk™ I don't have any idea what I'm doing.

But! To answer your question I would say, yes, that sounds correct, and I believe that is happening on lines `51` or `54` in the updated version.